Vagrantfile

```diff
@@ -70,10 +80,18 @@ Vagrant.configure('2') do |config|
   end
   config.ssh.forward_agent = true
   config.vm.synced_folder '.', '/vagrant', disabled: true
-  config.vm.synced_folder '.', "/home/vagrant/#{project_name}", type: 'nfs',
+  
+  if Vagrant::Util::Platform.windows? then
+	# Windows compatible mount options. Tested on Win 7 SP1
+    config.vm.synced_folder '.', "/home/vagrant/#{project_name}", type: 'nfs',
+    mount_options: ['dmode=775','fmode=775'],
+    linux__nfs_options: ['rw', 'no_subtree_check', 'all_squash', 'async']
```

Comment:
`config.vm.synced_folder '.', "/home/vagrant/#{project_name}", type: 'nfs',` 
and 
`linux__nfs_options: ['rw', 'no_subtree_check', 'all_squash', 'async']` 
are common and can be moved out of condition

Reply:
Updated